rescript/extract_seq_segments.py

```diff
@@ -49,5 +50,7 @@ def extract_seq_segments(input_sequences: DNAFASTAFormat,
                '--notmatched', no_match_f.name]
         if min_seq_len:
             cmd += ['--minseqlength', '%i' % min_seq_len]
+        if max_seq_len:
+            cmd += ['--maxseqlength', '%i' % max_seq_len]
```

Comment:
Also these if statements could be removed and the parameters could be passed with all the others with the change above.